src/CMakeLists.txt

```diff
@@ -73,6 +73,9 @@ add_library(slicot SHARED
 
 target_link_libraries(slicot lapack_aux ${ExtLibs})
 
+set_property(TARGET slicot PROPERTY VERSION "0.0.0")
+set_property(TARGET slicot PROPERTY SOVERSION 0 )
+
 install(TARGETS slicot
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib)
+        LIBRARY DESTINATION ${LIB_INSTALL_DIR}
```

Comment:
with the addition of these two lines, my `cmake ../` command from the build directory fails to create the builder. maybe this is a problem in my configuration, I do not know. I will look into it at the weekend more thoroughly. thanks for your attempt to port this to Fedora --- I am not that knowledgable in `cmake`.